site/content/docs/user/quick-start.md

```diff
@@ -232,14 +231,64 @@ to do that first before opening "Settings"
 <img src="/docs/user/images/docker-pref-1-win.png"/>
 
 Now, go to the **Advanced** settings page, and change the
-settings there, see [changing Docker's resource limits][Docker resource lims].  
+settings there, see [changing Docker's resource limits][Docker resource lims].
 
 <img src="/docs/user/images/docker-pref-build-win.png" alt="Setting 8Gb of memory in Docker for Windows" />
 
 
 You may also try removing any unused data left by the Docker engine - e.g.,
 `docker system prune`.
 
+## Customizing node images
```

Comment:
I would like to stop making this document longer. this is not quick-start material, most users should not ever do this.

Reply:
I've moved it to its own section